crowbar_framework/app/models/node_object.rb

```diff
@@ -1143,21 +1147,26 @@ def net_rpc_cmd(cmd)
     when :power_cycle
       unless system("net", "rpc", "shutdown", "-f", "-r", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
         Rails.logger.warn("samba command \"#{cmd}\" for #{@node.name} failed - node in unknown state")
+        [422, "Node in unknown state"]
       end
     when :power_off
       unless system("net", "rpc", "shutdown", "-f", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
         Rails.logger.warn("samba command \"#{cmd}\" for #{@node.name} failed - node in unknown state")
+        [422, "Node in unknown state"]
       end
     when :reboot
       unless system("net", "rpc", "shutdown", "-r", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
         Rails.logger.warn("samba command \"#{cmd}\" for #{@node.name} failed - node in unknown state")
+        [422, "Node in unknown state"]
       end
     when :shutdown
       unless system("net", "rpc", "shutdown", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
         Rails.logger.warn("samba command \"#{cmd}\" for #{@node.name} failed - node in unknown state")
+        [422, "Node in unknown state"]
       end
     else
       Rails.logger.warn("samba command #{cmd} failed for #{@node.name}.")
+      [422, "Command #{cmd} failed"]
```

Comment:
Actually, the log was wrong and your text is therefore wrong. It's more like "unknown command".